.github/workflows/examples-cc13x2x7_26x2x7.yaml

```diff
@@ -79,5 +78,102 @@ jobs:
                   path: |
                       .environment/gn_out/.ninja_log
                       .environment/pigweed-venv/*.log
-                      
 
+            - name: Build examples
+              timeout-minutes: 100
+              run: |
+                  scripts/run_in_build_env.sh "\
```

Comment:
do we really need these many configuration tests? This will add another hour(s) of CI time for every PR.